src/lib/player.ts

```diff
@@ -73,9 +73,12 @@ export const startCurrentPlayer = async () => {
 
 	// create buffer and load next song
 	const queueBuffer = new Tone.ToneAudioBuffer();
-	await queueBuffer.load(createApiSongUrl(songInQueue, gamestate));
-
-	playerStore.update({ queue: { ...queue, buffer: queueBuffer, song: songInQueue } });
+	try {
+		await queueBuffer.load(createApiSongUrl(songInQueue, gamestate));
```

Comment:
this can be shortened into:
`await queueBuffer.load(createApiSongUrl(songInQueue, gamestate)).catch(showErrorToast)`

Reply:
Wouldn't that update the state and enable the button?

Reply:
i mean instead of the try catch you can directly catch the Promise since `playerStore.update` is not  throwing

Reply:
You mean:
```Typescript
await queueBuffer.load(createApiSongUrl(songInQueue, gamestate)).catch(showErrorToast());
playerstore.update(...);
````
But then the playerstore will be updated, despite the song could not be loaded and the user could continue. That was my point. However, as is think about it, users should be able to use the app, even when the songs could not be loaded.